### DISCUSSION TITLE
Nouvelle île - l'île de glace

### DISCUSSION THREAD

**Discussion by Ravenclaw-l** [Crownicles/Crownicles#2582] [Idées acceptées / déjà implémentées]
Hello,

Je fais suite à une discussion sur le général. 
Après avoir potassé le sujet ce week-end, voici mes idées (à compléter, modifier, améliorer, je suis ouverte à toutes suggestions).

Pour la différencier de la première, ce serait une île de glace. Accessible depuis les points d'eau à proximité du Mont Célestrum par exemple.

Lieux et boss :

1 :evergreen_tree: forêt hurlante
- 1er boss : un loup (ou autre, genre un ours)
exemple d'attaques :
:tooth:  morsure
:eyes: traque silencieuse (je l’imagine en alté comme la protection du fantassin)

2 :ocean:  lac gelé
- 2ème boss : un spectre (un boss plus psychologique que physique)
exemple d'attaques :
:snowflake: toucher glacial (ralentit en même temps qu’il enlève de l’énergie)
:cyclone: cauchemars (alté qui plongerait le joueur dans un état de confusion)

3 :mountain_snow: montagne de neige
- 3ème boss : le yéti
exemples d'attaques :
:cold_face: coup de poing glacé
:bricks: mur de glace (ouais un peu comme Mei dans Overwatch, je m’inspire comme je peux)
:boom: rugissement

4 :snowflake: glacier éternel
- 4ème boss : le titan de glace
attaques :
:mount_fuji: avalanche
🥶 congélation (alté empêchant le joueur de jouer pendant un tour)
:scream:   manque d’Oxygène (enlève l’intégralité du souffle du joueur)
Objet spécial :
Pour contrer le manque d’oxygène, j’ai eu l’idée de créer un objet du type "bouteille d’oxygène" qui permettrait de garder le souffle intact. Cet objet se rechargerait très lentement entre chaque voyage sur l'île. Il faudra donc bien vérifier le niveau d'oxygène dans la bouteille avant chaque combat contre le titan.

Il faudrait également, à mon sens, enlever l'alté "gelé" du drain de chaleur du titan de magma, car elle correspond mieux au titan de glace.

N'hésitez pas à me dire ce que vous en pensez.

**Comment by Procion06** [comment]
Très bonne idée l'île de glace !

je verrai bien une île uniquement accessible aux top joueurs (lvl 100+)
plus longue avec 6 boss à combattre au lieu de 4
8 boss au total (avec 2 embranchements sur le parcours comme sur la 1ère île où tu dois choisir entre le village en ruine ou la mine brumeuse)
si le joueur passe le 4ème boss il atteint la cascade de vie où il régénère toute son énergie
il peut ensuite partir affronter les 2 derniers boss qui sont les plus puissants

**Reply by Ravenclaw-l** [reply]
La cascade de vie c'est une idée de génie.

**Comment by Ntalcme** [comment]
J'adore l'idée mais on pourrait faire les deux îles au moins une fois gratuitement par semaine ?

**Reply by Ravenclaw-l** [reply]
Tel que je l'imagine, l'île ne serait accessible qu'une fois par semaine et gratuitement.

**Reply by Ravenclaw-l** [reply]
Je veux dire en plus de l'île actuelle.

**Comment by Voisin1234** [comment]
salut

j’adore l’idée d’ouvrir une nouvelle île ça apporterait vraiment du neuf au jeu mais j’ai quelques questions pour mieux comprendre comment ça fonctionnerait

comment se passerait le trajet pour aller sur cette nouvelle île est-ce qu’on doit prendre un bateau comme pour l’île précédente ou bien un autre moyen

Est-ce qu’il y aura trois trajets différents à gérer entre les différentes îles ou tout serait centralisé?

Concernant les points est-ce qu’ils seront cumulés dans le classement des guildes pour cette nouvelle île ou y aura-t-il un classement spécifique?

Comme l’accès est réservé aux niveaux 100 que se passe-t-il si un membre de la guilde n’a pas encore atteint ce niveau est-ce qu’il devra rester sur les autres îles ou pourra-t-il quand même rejoindre ses camarades d’une manière ou d’une autre

**Reply by Ravenclaw-l** [reply]
Hello, perso, je ne suis pas forcément pour un accès réservé uniquement aux meilleurs joueurs, peut-être aux niveaux 80+, mais même là, j'émets des réserves, car ça risque de frustrer les joueurs.
Concernant l'accès, un voyage en bateau me semble adapté, vu qu'il s'agit d'une île. 
Les points de guilde devraient se cumuler au classement actuel seulement si l'accès est ouvert à tous les joueurs, sinon, ce serait injuste pour les guildes avec des membres de plus bas niveau.

**Comment by Feiryn** [comment]
Il y a déjà un début de l'île de glace qui est dans le code mais elle n'est pas terminée.
Il y a également déjà plusieurs îles qui peuvent être supportées par le code. Tous les joueurs vont sur la même île la même semaine, et l'île en question tourne chaque semaine

**Reply by Ravenclaw-l** [reply]
tu veux dire que l'île changerait chaque semaine ? semaine paire : île noire, semaine impaire: île de glace ?

**Reply by Cyyrene** [reply]
Ohh, ça serait trop bien d'avoir du changement comme ça chaque semaine

**Reply by Feiryn** [reply]
> tu veux dire que l'île changerait chaque semaine ? semaine paire : île noire, semaine impaire: île de glace ?

Oui, et à partir de 3 cartes, on passe sur du random mais sans répétition (pas 2x la même île d'affilé)

**Reply by Ravenclaw-l** [reply]
> > tu veux dire que l'île changerait chaque semaine ? semaine paire : île noire, semaine impaire: île de glace ?
> 
> Oui, et à partir de 3 cartes, on passe sur du random mais sans répétition (pas 2x la même île d'affilé)

J'aime beaucoup l'idée de ne pas savoir quelle île sera disponible la semaine suivante.

**Reply by Procion06** [reply]
bof, ça manque de contenu end game
remplacer une île par une autre ça n'ajoute pas grand chose au final
avoir la possibilité de faire 2 îles différentes chaque semaine ça c'est bien :)

**Reply by Ravenclaw-l** [reply]
Oui, c'est idéal pour 2 îles. À partir de 3, il sera difficile de rassembler toute la guilde pour toutes les explorer. 
L'avantage d'avoir toutes les îles à disposition est qu'on pourra choisir laquelle visiter, mais avec un système aléatoire, la surprise serait totale. Les deux options se valent, j'aurais du mal à trancher.

**Reply by Voisin1234** [reply]
![image](https://github.com/user-attachments/assets/9afe6880-a582-432a-98ed-0301cce3d00b) en plus sa donne déjà envie juste de lire de def

**Reply by Ravenclaw-l** [reply]
> ! en plus sa donne déjà envie juste de lire de def

Merci, je viens de trouver dans le code l'ébauche de l'île de glace. Si l'idée de développer cette île est acceptée, on pourra utiliser les idées déjà existantes, qui sont plutôt pas mal.

**Comment by kirit000** [comment]
Reven je t'aime t'as mis le Yéti 🥹

**Reply by Ravenclaw-l** [reply]
ton idée était bonne :thumbsup:

**Comment by pannecrepe** [comment]
Je propose de rajouter un nouveau mini évent sur l'île de glace qui permettrait de nous faire changer de trajet et de passer a l'autre trajet du croisement sous prétexte qu'une avalanche nous a bloqué le passage ( sa ne nous mettrais pas d'altération d'état mais rajouterai du temps de trajet ) .

**Reply by Ravenclaw-l** [reply]
Le changement de trajectoire est une bonne idée. 

Perso, je sais que faire l'île me prend généralement 30 min max, et j'organise mes dispo en fonction de ce temps. Si tu m'ajoutes 30 min supplémentaires, ça me compliquerait les choses. Peut-être limiter le temps de trajet sup à 15 min max ?
Voir ce que les autres en pensent.

**Reply by pannecrepe** [reply]
Moi je pensais à 5 min pour pas que se soit trop abusé mais pas trop nul non plus
